### PR TITLE
Feature #114075 feat: Add new field type in tjfield textareacounter

### PR DESCRIPTION
**Pull request by vaibhavsTekdi**
Using this field user set max length, min length

### CODE REVIEW

File: admin_language/en-GB/en-GB.com_tjfields.ini [hunk 1]
```diff
@@ -385,3 +385,12 @@ COM_TJFIELDS_PERMISSION_EDIT_FIELD_VALUE_DESC="Allows users in the group to edit
 COM_TJFIELDS_PERMISSION_EDIT_OWN_FIELD_VALUE="Edit Own Field Value"
 COM_TJFIELDS_PERMISSION_EDIT_OWN_FIELD_VALUE_DESC="Allows users in the group to edit own data for the field"
 COM_TJFIELDS_FILE_DOWNLOAD="Download"
+
+;textareacounter field
+COM_TJFIELDS_TEXTAREACOUNTER="Textareacounter"
+COM_TJFIELDS_FORM_LBL_FIELD_COUNTERTEXT="Contertext"
+COM_TJFIELDS_FORM_DESC_FIELD_COUNTERTEXT="Format : Characters remaining {remaining} / {maxlength}"
+COM_TJFIELDS_FORM_LBL_FIELD_MAXLENGHT="Maxlength"
+COM_TJFIELDS_FORM_DESC_FIELD_MAXLENGHT="The maximum number of characters allowed in the text area"
```
Comment (ankush-maherwal) [inline admin_language/en-GB/en-GB.com_tjfields.ini]: text area => textarea

File: admin_language/en-GB/en-GB.com_tjfields.ini [hunk 2]
```diff
@@ -385,3 +385,12 @@ COM_TJFIELDS_PERMISSION_EDIT_FIELD_VALUE_DESC="Allows users in the group to edit
 COM_TJFIELDS_PERMISSION_EDIT_OWN_FIELD_VALUE="Edit Own Field Value"
 COM_TJFIELDS_PERMISSION_EDIT_OWN_FIELD_VALUE_DESC="Allows users in the group to edit own data for the field"
 COM_TJFIELDS_FILE_DOWNLOAD="Download"
+
+;textareacounter field
+COM_TJFIELDS_TEXTAREACOUNTER="Textareacounter"
+COM_TJFIELDS_FORM_LBL_FIELD_COUNTERTEXT="Contertext"
```
Comment (ankush-maherwal) [inline admin_language/en-GB/en-GB.com_tjfields.ini]: check spelling "Contertext"

File: administrator/models/fields/textareacounter.php [hunk 3]
```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * @version    SVN:<SVN_ID>
+ * @package    TJFields
+ * @author     Techjoomla <extensions@techjoomla.com>
+ * @copyright  Copyright (c) 2009-2016 TechJoomla. All rights reserved
```
Comment (ankush-maherwal) [inline administrator/models/fields/textareacounter.php]: check copyright year 
Update the file comments

File: admin_language/en-GB/en-GB.com_tjfields.ini [hunk 4]
```diff
@@ -385,3 +385,12 @@ COM_TJFIELDS_PERMISSION_EDIT_FIELD_VALUE_DESC="Allows users in the group to edit
 COM_TJFIELDS_PERMISSION_EDIT_OWN_FIELD_VALUE="Edit Own Field Value"
 COM_TJFIELDS_PERMISSION_EDIT_OWN_FIELD_VALUE_DESC="Allows users in the group to edit own data for the field"
 COM_TJFIELDS_FILE_DOWNLOAD="Download"
+
+;textareacounter field
+COM_TJFIELDS_TEXTAREACOUNTER="Textareacounter"
```
Comment (ankush-maherwal) [inline admin_language/en-GB/en-GB.com_tjfields.ini]: value of language constant is not readable

can we change this from "Textareacounter" => "Textarea Counter"

**Comment by vaibhavsTekdi** [comment]
All the changes & development done under previous MRs..

**Pull request closed without merging**